### ISSUE TITLE
Disable level up buttons while fetching new level up choices

### ISSUE THREAD

**Issue by Zenoo** [Zenoo/labrute#1069]

**Comment by Elmenax2** [comment]
@Zenoo what is the context for that one? Having the 2 "validate" buttons on the level up page when `getLevelUpChoices` is on-going?

**Comment by Zenoo** [comment]
The issue here is that after a user selects a level up choice, if the server is busy, the user might click again on the button, and if he has enough XP, it will trigger its next level, selecting the side he clicked on, without him having seen the option.
So the solution is to hide the validate buttons after a user clicks on any of them, and make them reappear once the request ends

**Comment by Elmenax2** [comment]
Would that make sense to bundle #585 with this one at the same time? I guess it would be nice for users to see on screen that something is happening hence the button being disabled

**Comment by Zenoo** [comment]
Yes, if #585 is implemented, this issue will be solved, since the loader will prevent any further user action.

**Comment by Elmenax2** [comment]
Okay then I can take care of this one and #585
